README.md

```diff
@@ -194,13 +194,19 @@ const App = ({ message }: AppProps): JSX.Element => <div>{message}</div>;
 
 // you can also inline the type declaration; eliminates naming the prop types, but looks repetitive
 const App = ({ message }: { message: string }) => <div>{message}</div>;
+
+// Alternatively, you can use `React.FunctionComponent` (or `React.FC`), if you prefer.
+// With latest React types and TypeScript 5.1. it's mostly a stylistic choice, otherwise discouraged.
```

Comment:
Does it make sense here to specify exact `@types/react` version, instead of referring to latest?

Reply:
```suggestion
// With latest React types (earliest is `@types/react@18.0.0`) and TypeScript 5.1. it's mostly a stylistic choice, otherwise discouraged.
```